sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-multi-database/src/main/java/com/azure/spring/sample/multi/database/MultiDatasourceApplication.java

```diff
@@ -0,0 +1,103 @@
+import javax.annotation.PreDestroy;
+import java.util.List;
+
+@SpringBootApplication
+public class MultiDatasourceApplication implements CommandLineRunner {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Autowired
+    private UserRepositoryForMYSQL userRepositoryForMYSQL;
+
+    @Autowired
+    @Qualifier("secondaryDatabaseTemplate")
+    private CosmosTemplate secondaryDatabaseTemplate;
+
+    @Autowired
+    @Qualifier("secondaryDatabase2Template")
+    private CosmosTemplate secondaryDatabase2Template;
+
+    @Autowired
+    @Qualifier("primaryDatabase2Template")
+    private ReactiveCosmosTemplate primaryDatabase2Template;
+
+    private final User user = new User("1024", "1024@geek.com", "1k", "Mars");
+    private UserForMYSQL user2;
+    private static CosmosEntityInformation<User, String> userInfo = new CosmosEntityInformation<>(User.class);
+
+    private final Book book = new Book("9780792745488", "Zen and the Art of Motorcycle Maintenance", "Robert M. Pirsig");
+    private static CosmosEntityInformation<Book, String> bookInfo = new CosmosEntityInformation<>(Book.class);
+
+    public static void main(String[] args) {
+        SpringApplication.run(MultiDatasourceApplication.class, args);
+    }
+
+    public void run(String... var1) throws Exception {
+
+        final List<User> users = this.userRepository.findByEmailOrName(this.user.getEmail(), this.user.getName()).collectList().block();
+        users.forEach(System.out::println);
+        users.stream().forEach(this::insertValueToMYSQL);
+
+        Book bookGet = primaryDatabase2Template.findById(Book.class.getSimpleName(), book.getIbsn(), Book.class).block();
+        System.out.println(bookGet);
+
+        User result = secondaryDatabaseTemplate.findById(User.class.getSimpleName(),
+            user.getId(), User.class);
+        System.out.println(result);
+        Book result2 = secondaryDatabase2Template.findById(Book.class.getSimpleName(),
+            book.getIbsn(), Book.class);
+        System.out.println(result2);
+
+        userRepositoryForMYSQL.findAll().forEach(System.out::println);
+    }
+
+    public void insertValueToMYSQL(User user){
+        user2 = new UserForMYSQL(user.getId(), user.getEmail(), user.getName(), user.getAddress());
+        userRepositoryForMYSQL.save(user2);
+    }
+
+    @PostConstruct
+    public void setup() {
+
+        secondaryDatabaseTemplate.createContainerIfNotExists(userInfo);
+        secondaryDatabaseTemplate.insert(User.class.getSimpleName(), user,
+            new PartitionKey(user.getName()));
+        secondaryDatabase2Template.createContainerIfNotExists(bookInfo);
+        secondaryDatabase2Template.insert(Book.class.getSimpleName(), book,
+            new PartitionKey(book.getName()));
+
+        primaryDatabase2Template.createContainerIfNotExists(bookInfo).block();
+        primaryDatabase2Template.insert(Book.class.getSimpleName(), book,
+            new PartitionKey(book.getName())).block();
```

Comment:
1. Write them in order: `primary` first, then `secondary`.
2. Do not break line if the line length does not exceed 120.
3. Do we need to add contents about `primaryDatabase1Template` here?

Reply:
``
3. Do we need to add contents about primaryDatabase1Template here?
``
primaryDatabase1Template is reactive, so we can use this.userRepository.save(user).block() instead of primaryDatabase1Template.insert()

Reply:
`primaryDatabase1Template.insert()` can work?

Then my suggestion is to write like this:
```
primaryDatabase1Template.insert() // Same to userRepository.save(user).block().
```